hushline/settings/__init__.py

```diff
@@ -800,4 +821,27 @@ async def alias(username_id: int) -> Response | str:
             profile_form=profile_form,
         )
 
+    @bp.route("/update-directory-intro", methods=["POST"])
+    @admin_authentication_required
+    def update_directory_intro_text() -> Response:
+        form = UpdateDirectoryIntroTextForm()
+        if form.validate_on_submit():
+            # Get the raw input from the form
```

Comment:
These should be collapsed into just two significant lines since the meaning of each is exceedingly clear. More lines and temp vars is more to keep track of mentally

```python
OrganizationSetting.upsert(
    key=OrganizationSetting.DIRECTORY_INTRO,
    value=Markup(sanitize_input(from.directory_intro_text.data)),
)
db.session.commit()
```